proxy/upgrade_test.go

```diff
@@ -288,8 +288,8 @@ func TestServeHTTP(t *testing.T) {
 				return
 			}
 			if ti.backendClosesConnection {
-				if pong != "HTTP/1.1 400 Bad Request\r\n" {
-					t.Errorf("wrong bad response <%s>", pong)
+				if pong != `HTTP/1.1 200 OK\r\n` {
```

Comment:
not 100% sure yet, why this changed.

Reply:
Hmm this looks wrong

Reply:
I did a bisect of Go (from go1.18.5 to go1.19) and found this commit to be the cause of it: https://github.com/golang/go/commit/770e0e584a98dfd5e8d0d00558085c339fda0ed7

Reply:
Resolved in https://github.com/zalando/skipper/pull/2105